jira/resources.py

```diff
@@ -911,6 +911,11 @@ def __init__(self, options, session, raw=None):
 
 # Utilities
 
+def setattrWithUnicode(obj, name, value):
```

Comment:
Why not “upgrading” setattr() instead?
I do not find adding a new method as something easy to maintain or not confusing.
if possible we should fix original method